### PR TITLE
issue: 1515087 Fix std::exception deprecated warnings

### PR DESCRIPTION
**Pull request by liranoz12**
Signed-off-by: Liran Oz <lirano@mellanox.com>

### CODE REVIEW

File: src/message.cpp [hunk 1]
```diff
@@ -69,15 +69,15 @@ MemException::MemException(const char *file, int line, const char *handler, size
...
+/*static*/ void Message::initMaxSeqNo(uint64_t seqno) {
     if (ms_maxSequenceNo)
         throw std::logic_error("MaxSeqNo is already initialized");
     else
         ms_maxSequenceNo = seqno;
 }
 
 //------------------------------------------------------------------------------
-Message::Message() throw(std::exception) { // throws in case fatal error occurred
+Message::Message() { // throws in case fatal error occurred
```
Comment (igor-ivanov) [inline src/message.cpp]: Please remove useless comment in all identical places

**Comment by liranoz12** [comment]
bot:retest

**Comment by liranoz12** [comment]
@igor-ivanov done

**Review by igor-ivanov — APPROVED** [review]
:+1:

**Comment by igor-ivanov** [comment]
Pushed

**Pull request closed without merging**